CHANGES.md

```diff
@@ -1,3 +1,12 @@
+# vob 3.0.6 (2025-08-14)
+
+* Transparently optimise `iter_{set|unset}_bits(...).count()` to use platform
+  "count bits" directly, rather than forcing the full iterator to execute.
+  Depending on your machine, this can lead to savings of 2-3 orders of magnitude.
+
```

Comment:
I don't know if it worth mentioning, but we updated the 'normal' interator.count() too.
It now does math instead of exhausting the iterator (`range.count()`)